docs/content/src/references/references.md

```diff
@@ -3,18 +3,8 @@
 ## Ballerina Central
 [Ballerina Central](https://central.ballerina.io/) is a publicly accessible directory that can be used to share your local modules with a wider community.
 
-Following are the list of EI Connectors that are available in the Ballerina Central.
-
-* [JMS](https://central.ballerina.io/wso2/jms)
-* [Samba](https://central.ballerina.io/wso2/smb)
-* [FTP](https://central.ballerina.io/wso2/ftp)
-* [Amazon SQS](https://central.ballerina.io/wso2/amazonsqs)
-* [Amazon S3](https://central.ballerina.io/wso2/amazons3)
-* [Salesforce](https://central.ballerina.io/wso2/sfdc46)
-* [Google Spreadsheets](https://central.ballerina.io/wso2/gsheets4)
-* [Gmail](https://central.ballerina.io/wso2/gmail)
-* [SOAP](https://central.ballerina.io/wso2/soap)
-* [Store-Forward](https://central.ballerina.io/wso2/storeforward)
+## EI Connectors in Ballerina Central
+The list of EI Connectors can be found [here](https://github.com/wso2/ballerina-integrator/blob/master/docs/content/src/references/ei-connectors.md).
```

Comment:
We should have a relative link here. Something like ../ei-connectors/ I think. Otherwise this like will point to the repo and we need it to point to the doc site.

Reply:
Done